jetstream/cli.py

```diff
@@ -809,18 +832,21 @@ def export_experiment_logs_to_json(
     ctx, log_project_id, log_dataset_id, log_table_id, bucket, experiment_slug, project_id, date
 ):
     """Export all error logs for this experiment as JSON to a GCS bucket."""
-    for slug in experiment_slug:
-        export_experiment_logs(
-            project_id,
-            bucket,
-            slug,
-            log_project_id,
-            log_dataset_id,
-            log_table_id,
-            date,
-            None,
-            ctx.obj["log_config"],
-        )
+    if bucket is None:
+        click.echo("No bucket specified. Logs results won't be exported to GCS.")
```

Comment:
Same thoughts here: should we use `logger` instead of `click.echo`? And should we raise a `ValueError` because this doesn't make sense without the bucket?